BridgeAppSDK/SBAUserWrapper+Bridge.swift

```diff
@@ -634,15 +634,18 @@ extension NSDictionary: SBAUserSessionInfoWrapper {
 
 extension SBBUserSessionInfo: SBAUserSessionInfoWrapper {
     var createdOn: Date {
+        guard self.studyParticipant != nil && self.studyParticipant.createdOn != nil else { return Date() }
         // If the participant hasn't signed in yet, there won't be a createdOn date in the placeholder study participant object.
```

Comment:
Would this work:
 return self.studyParticipant?.createdOn? ?? Date()
?
If not, is the ?? Date() still needed on the return statement?